etl-pipeline/api/app.py

```diff
@@ -63,27 +71,25 @@ def run(self):
         else:
             serve(self.app, host="0.0.0.0", port=80)
 
-    def createErrorResponses(self):
+    def create_error_responses(self):
         @self.app.errorhandler(404)
-        def pageNotFound(error):
-            logger.warning("Page not found")
-            logger.debug(error)
+        def page_not_found(error):
+            logger.exception("Page not found")
```

Comment:
logger.exception also logs the error. 

we don't usually use debug logs - removing